src/Room.js

```diff
@@ -258,6 +258,25 @@ class Room extends EventEmitter {
     door.locked = true;
   }
 
+/**
+  * Get the detail descriptions contained in a room by id
+  * @param {string} id    detail description id
+  * @return {false|Object}
+  */
+  findDetailDesc(detailDescId) {
+    const detailDescs = Array.from(this.detailDescs).filter(dd => dd.id.indexOf(detailDescId) === 0);
+
+    if (!detailDescs.length) {
```

Comment:
Could these two conditionals be condensed into a single one such as `if (detailDescs.length !== 1)`?